tests/support/fixtures.ts

```diff
@@ -139,7 +139,10 @@ export const test = base.extend<{
     await Fs.mkdir(stateDir, { recursive: true });
 
     await use(stateDir);
-    if (process.env.CI && testInfo.status === "passed") {
+    if (
+      process.env.CI &&
+      (testInfo.status === "passed" || testInfo.status === "skipped")
```

Comment:
When does this happens? Are we at some point skipping visual tests?
`testInfo.status === "skipped"`

Reply:
No, this is to eliminate an empty `test.log` being uploaded as an artifact, if we skip a test that's only supposed to run on one browser. See `clipboard.spec.ts`.